geoopt/manifolds/stereographic/manifold.py

```diff
@@ -418,6 +449,53 @@ def random_normal(
 
     random = random_normal
 
+    def wrapped_normal(
```

Comment:
Did not find a coverage test for that

Reply:
We also need a scaling rule for std
```suggestion
@__scaling__(ScalingInfo(std=-1), "random")
    def wrapped_normal(
```